lib/mix/test/mix/tasks/deps_test.exs

```diff
@@ -251,6 +251,12 @@ defmodule Mix.Tasks.DepsTest do
     end
   end
 
+  test "fails with message on missing dependencies" do
+    assert_raise Mix.Error, ~r/"mix deps\.unlock" expects dependencies as arguments/, fn ->
```

Comment:
@gmile it looks like the Travis failure is legit here. You probably need to setup the project where this command is going to be run in (because of the call to `Mix.Project.get!/0` at the top of `Mix.Tasks.Deps.Unlock.run/1`), like we do in other tests here:

```elixir
Mix.Project.push DepsApp
in_tmp context.test, fn ->
  assert_raise ...
end